causalml/inference/meta/xlearner.py

```diff
@@ -641,12 +641,6 @@ def fit(self, X, treatment, y, p=None):
         else:
             check_p_conditions(p, self.t_groups)
 
```

Comment:
Thanks @paullo0106 for editing! Quick question, any reason we are dropping line 644 - 649?

Reply:
yeah so from the code `p` is not being used in `fit()`, it's used in `predict()` and `estimate_ate()` when estimating average treatment effect - this is how the X-learner algorithm works, right?